core/ordering/cosipbft/mod.go

```diff
@@ -223,9 +231,9 @@ func NewService(param ServiceParam, opts ...ServiceOption) (*Service, error) {
 		actor:                    actor,
 		val:                      param.Validation,
 		verifierFac:              param.Cosi.GetVerifierFactory(),
-		timeoutRound:             RoundTimeout,
-		timeoutRoundAfterFailure: RoundTimeout,
-		timeoutViewchange:        RoundTimeout,
+		timeoutRound:             DefaultRoundTimeout,
+		timeoutRoundAfterFailure: DefaultRoundTimeoutAfterFailure,
+		timeoutViewchange:        DefaultTimeoutBeforeViewchange,
```

Comment:
do we really want to change these values in production?
It seems like a rather impactful change.